pkg/crc/config/settings.go

```diff
@@ -91,6 +92,8 @@ func RegisterSettings(cfg *Config) {
 		"Enable experimental features (true/false, default: false)")
 	cfg.AddSetting(EmergencyLogin, false, ValidateBool, SuccessfullyApplied,
 		"Enable emergency login for 'core' user. Password is randomly generated. (true/false, default: false)")
+	cfg.AddSetting(PersistentVolumeSize, constants.DefaultPersistentVolumeSize, validatePersistentVolumeSize, SuccessfullyApplied,
+		fmt.Sprintf("Total size in GiB of the persistent volume used by the CSI driver for %s preset (must be greater than or equal to '%d')", preset.Microshift, constants.DefaultPersistentVolumeSize))
```

Comment:
ahh, sorry i forgot to mention this earlier, maybe we should use `preset.Microshift.ForDisplay()` here

Reply:
@anjannath since preset have string method `%s` should able to get preset and Display is for end user display I think here they should know the `preset` value instead of Display?